docs/src/Sheet.doc.js

```diff
@@ -1,17 +1,24 @@
 // @flow strict
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
```

Comment:
Don't forget the feedback callout! We might actually get some real feedback on this new sandbox approach 🤞

Reply:
oh good point!